source/_docs/scripts/conditions.markdown

```diff
@@ -396,25 +396,25 @@ condition:
   after_offset: "-01:00:00"
 ```
 
-This is 'when dark' - equivalent to a state condition on `sun.sun` of `below_horizon`:
+This is 'when light' - equivalent to a state condition on `sun.sun` of `above_horizon`:
 
 ```yaml
 condition:
   - condition: sun
-    after: sunset
-    before: sunrise
+    after: sunrise
+    before: sunset
 ```
 
-This is 'when light' - equivalent to a state condition on `sun.sun` of `above_horizon`:
+This is (invalid) 'when dark' - equivalent to a state condition on `sun.sun` of `below_horizon`:
 
 ```yaml
 condition:
   - condition: sun
-    after: sunrise
-    before: sunset
+    after: sunset
+    before: sunrise
 ```
 
-We cannot use both keys in this case as it will always be `false`.
+However, we cannot use both keys in this case as it will always be `false`, so instead you have to use:
```

Comment:
This is actually a bug, this part of the documentation should be removed, as it is not the intended functionality.